lib/components/FilePicker/FileListRow.vue

```diff
@@ -47,9 +48,14 @@ const emit = defineEmits<{
 }>()
 
 /**
- * The displayname of the current node
+ * The displayname of the current node (excluding file extension)
  */
-const displayName = computed(() => props.node.attributes?.displayname || props.node.basename)
+const displayName = computed(() => props.node.attributes?.displayname || props.node.basename.slice(0, props.node.extension ? -props.node.extension.length : undefined))
```

Comment:
```suggestion
const displayName = computed(() => props.node.attributes?.displayName || props.node.basename.slice(0, props.node.extension ? -props.node.extension.length : undefined))
```

https://github.com/nextcloud/server/blob/24af920992ba668a90bb71d5a2bce84109a6587e/apps/files/src/components/FileEntry.vue#L323-L336